### ISSUE TITLE
Default image on report submission appears unexpectedly

### ISSUE THREAD

**Issue by lmcnulty** [responsible-ai-collaborative/aiid#635]
In addition to #624, there are some UX issues with submitting a report
The default report image doesn't show up until after you unfocus the image url input field. This is awkward as it's very large and pushes the other elements out of the way. It's also unlabelled, so it's not necessarily clear what happened. This is especially jarring when using the tab key to advance to the next field, as shown in the video below:

  https://user-images.githubusercontent.com/25443411/169911491-8bf3566d-7671-48a1-a5b3-31017dafd02e.mp4

I think the default image should be small, labelled, and visible on the initial page load.

**Comment by smcgregor** [comment]
Agreed. I think shrinking the image and having the placeholder displayed until there is a new candidate image sounds good to me.
